packages/core/src/node/file-uri.ts

```diff
@@ -54,6 +54,9 @@ export namespace FileUri {
                     return fsPathFromVsCodeUri + '\\';
                 }
             }
+            if (fsPathFromVsCodeUri.startsWith('/file:')) {
```

Comment:
It looks weird, fsPath should not have scheme already. How can I get URI leading to such case. I think some clients are likely to be bogus.

Reply:
No idea, I reused the previous changes.

Reply:
@mmorhun Could you give a hint? It looks like there is a bug in how some client constructs URI. We should not expect such fs paths.

Reply:
let's remove it, if there will be a regression, we will fix it properly.

Reply:
@akosyakov I added that code as a quick fix. I agree that this is a kludge and we have to find proper solution. The reason I added it was that `FileUri.fsPath`, was returning wrong path: `/file:/right/path`.

Reply:
@mmorhun it can happen only if a uri passed to `FileUri.fsPath` is bogus, the real issue is to figure how one get such bogus URI and fix the producer